src/main/java/io/github/dftrakesh/cloverrest/model/updateInventory/updateStockResponse.java

```diff
@@ -0,0 +1,17 @@
+package io.github.dftrakesh.cloverrest.model.updateInventory;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class updateStockResponse {
```

Comment:
change name updateStockResonse to "InventoryResponse"
**Note: class name always First letter capital**

Reply:
changed  name